docker/env/version

```diff
@@ -1 +1 @@
-0.40
+0.43
```

Comment:
why did you move 3 versions up?

Reply:
cause seem like we use those number before in some PR, and i'm failing in CI cause of that. (i.e. cause of images already there)